src/rust/engine/process_execution/src/remote.rs

```diff
@@ -814,12 +814,14 @@ pub fn make_execute_request(
     mut platform_properties,
   } = metadata;
 
-  if !req.append_only_caches.is_empty() {
-    platform_properties.extend(NamedCaches::platform_properties(
-      &req.append_only_caches,
-      &cache_key_gen_version,
-    ));
-  }
+  // TODO: Disabling append-only caches in remoting until server support exists due to
```

Comment:
There is one test that will need updating as well.

Reply:
Tests pass for `cargo test` in process_execution crate. I did have to push a commit for lint fix.